packages/data-store/src/__tests__/contact.entities.test.ts

```diff
@@ -544,32 +551,11 @@ describe('Database entities tests', (): void => {
     await expect(dbConnection.getRepository(IdentityEntity).save(identityEntity)).rejects.toThrowError('Blank metadata labels are not allowed')
   })
 
-  it('should throw error when saving identity with blank metadata value', async (): Promise<void> => {
```

Comment:
Redundant because saving an empty metadata fields no longer throws an error. 
As discussed on Slack, DB insertion is skipped when the metadata value is empty.